### ISSUE TITLE
Build Error with AFNetworking 3.0.x

### ISSUE THREAD

**Issue by ecoopnet** [kazu0620/SRGVersionUpdater#4]
With AFNetworking ver 3.0.4,
this library raises build error that
`'AFHTTPRequestOperationManager.h' is not found`
in SRGVersionUpdater.m:

```
#import "AFHTTPRequestOperationManager.h"
```

This is because of AFNetworking changes:
https://github.com/AFNetworking/AFNetworking/wiki/AFNetworking-3.0-Migration-Guide

```
Classes Removed

The following classes have been removed from AFNetworking 3.0:

AFURLConnectionOperation
AFHTTPRequestOperation
AFHTTPRequestOperationManager
```

**Comment by vijaytholpadi** [comment]
Hello,

Have fixed this in PR #5 

Thanks.

**Comment by kazu0620** [comment]
thanks a lot @vijaytholpadi :smile:  i merged #5.
